### PR TITLE
Musician profile persistence #297

### LINKED ISSUE #297: Musician Profile not Persisting
The musician profile information is not persisting after save.  Form fields may also not be correct.

### PR DESCRIPTION
**Pull request by fredmon3**
This PR closes #297 

#### What does this PR do?

Adds persistence of the musician's profile.

### CODE REVIEW

File: client/src/containers/MusicianProfileContainer.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+function urlFn(state) {
+  return `users/${getUserId(state)}`;
+}
+
+const mapDispatchToProps = {
+  updateUser
+};
+
+function getInitialValues(state) {
```
Comment (wdoug) [inline client/src/containers/MusicianProfileContainer.js:15]: Doesn't `gimmeData` already give this to us (just passed in as a `data` prop)?

File: client/src/components/MusicianProfile.js [hunk 2]
```diff
@@ -62,17 +61,16 @@ const MusicianProfile = ({ handleSubmit }) => (
 
         <FormattedFormField title="Instrument/Voice">
           <Field
-            name="majorInstrument"
+            name="instruments[0]"
```
Comment (wdoug) [inline client/src/components/MusicianProfile.js:64]: Wow you can do that?

Comment (fredmon3) [reply client/src/components/MusicianProfile.js:64]: I was fairly surprised it worked, too. 😆

File: client/src/containers/MusicianProfileContainer.js (hunk 1 above)
Comment (fredmon3) [reply client/src/containers/MusicianProfileContainer.js:15]: I believe this is the correct way to supply initial values for form defaults to redux form.

**Review by wdoug — APPROVED** [review]
Looks good

**Pull request merged**